src/hl/group.rs

```diff
@@ -167,6 +169,36 @@ impl Group {
         let name = to_cstring(name)?;
         Dataset::from_id(h5try!(H5Dopen2(self.id(), name.as_ptr(), H5P_DEFAULT)))
     }
+
+    /// Non-recursively gets all the members in the group
+    pub fn get_member_names(&self) -> Result<Vec<String>> {
+        extern "C" fn get_members_callback(_id: hid_t,
+                                           name: *const c_char,
+                                           _info: *const H5L_info_t,
+                                           op_data: *mut c_void) -> herr_t {
+            let other_data: &mut Vec<String> = unsafe { &mut *(op_data as *mut Vec<String>) };
+
+            let c_str: &CStr = unsafe { CStr::from_ptr(name) };
+            let str_slice: &str = c_str.to_str().unwrap();
```

Comment:
There's `string_from_cstr()` which should be already imported in the prelude which woudl make this simpler

Reply:
Thanks for the suggestion. Done.